descriptors/ModuleDescriptor-template.json

```diff
@@ -657,7 +685,7 @@
     },
     {
       "id": "patron-notice-policy-storage",
-      "version": "0.7"
+      "version": "2.0"
```

Comment:
I imagine this will be amended once the the conversation [on](https://github.com/folio-org/mod-circulation-storage/pull/186) are resolved.

Reply:
Changed the version to 0.9 (left an interface experimental).